python_requirements.txt

```diff
@@ -1,19 +1,37 @@
-certifi==2020.6.20
-chardet==3.0.4
-cycler==0.10.0
-idna==2.10
-kiwisolver==1.2.0
-matplotlib==3.3.0
-numpy==1.19.1
-pandas==1.1.0
-Pillow==7.2.0
-pyparsing==2.4.7
-python-dateutil==2.8.1
-pytz==2020.1
-requests==2.24.0
-scipy==1.5.2
-seaborn==0.10.1
-six==1.15.0
-tabulate==0.8.7
-urllib3==1.25.10
-wcwidth==0.2.5
+asn1crypto==0.24.0
```

Comment:
Had to update all libraries and take a `pip freeze` snapshot because, for some reason, `pip install -r python_requirements.txt` failed with existing ones (likely because system python3 is now 3.10 not 3.7)